src/latexify/config.py

```diff
@@ -0,0 +1,67 @@
+"""Definition of the Config class."""
+
+from __future__ import annotations
+
+import dataclasses
+
+from typing import Any
+
+
+@dataclasses.dataclass(frozen=True)
+class Config:
+    """Configurations to control the behavior of latexify."""
```

Comment:
Could you put descriptions of members here?

```suggestion
    """Configurations to control the behavior of latexify.
    
    Attributes:
        foo: ...
        bar: ...
    """
```